Apps/W1/Shopify/app/src/Customer/Codeunits/ShpfyCustByBillto.Codeunit.al

```diff
@@ -35,8 +35,8 @@ codeunit 30111 "Shpfy Cust. By Bill-to" implements "Shpfy ICustomer Mapping"
     internal procedure DoMapping(CustomerId: BigInteger; JCustomerInfo: JsonObject; ShopCode: Code[20]; TemplateCode: Code[10]; AllowCreate: Boolean): Code[20];
```

Comment:
this function could really use being split into several smaller bits. It's very long and does multiple things